from opal.core.exceptions import Error


class PatientNotFoundError(Error):
    pass
